package.json

```diff
@@ -23,12 +25,26 @@
   },
   "homepage": "https://github.com/cubbit/interns-training#readme",
   "devDependencies": {
+    "@commitlint/cli": "^13.1.0",
+    "@commitlint/config-conventional": "^13.1.0",
     "@types/chai": "^4.2.15",
     "@types/mocha": "^8.2.2",
     "@types/node": "^12.12.11",
     "@types/sinon": "^9.0.11",
     "chai": "^4.3.4",
+    "cz-conventional-jira-smart-commit": "0.1.1",
+    "husky": "^7.0.0",
     "mocha": "^8.3.2",
     "sinon": "^10.0.0"
+  },
+  "config": {
+    "commitizen": {
+      "path": "./node_modules/cz-conventional-jira-smart-commit"
+    }
+  },
+  "commitlint": {
+    "extends": [
+      "@commitlint/config-conventional"
+    ]
   }
```

Comment:
:mage_man:

Reply:
Should I update the whole package.json to comply with allman style?

Reply:
Oh no, this emoji means that you did great!